README.md

```diff
@@ -183,8 +182,7 @@ let searchStrings = textField.rac_textSignal()
     .throttle(0.5, onScheduler: QueueScheduler.mainQueueScheduler)
 ```
 
-This prevents values from being sent less than 0.5 seconds apart, so the user
-must stop editing for at least that long before we’ll use their search string.
+This will result in the values being sent _every_ 0.5 seconds after the first event, so the user must stop editing for at least that long before we'll use their search string.
```

Comment:
This isn't really accurate. Values aren't sent every 0.5 seconds. That part was accurate before—it "prevents values from being sent less than 0.5 seconds apart". And the user doesn't need to stop editing before we'll use their search string—it means we'll use their search string at most every 0.5 seconds, and we'll get the final value at most 0.5 seconds after they type it.

Here's the documentation for `throttle`:

```swift
	/// Throttle values sent by the receiver, so that at least `interval`
	/// seconds pass between each, then forwards them on the given scheduler.
	///
	/// If multiple values are received before the interval has elapsed, the
	/// latest value is the one that will be passed on.
	///
	/// If the input signal terminates while a value is being throttled, that value
	/// will be discarded and the returned signal will terminate immediately.
```